aws.md

```diff
@@ -0,0 +1,73 @@
+
+![Choose Instance details](images/aws/launch-3.png)
+
+Add storage. The AWS EC2 launch wizard will suggest some form of storage by default. For our testing purposes this should be fine, but if you know that you need more storage, or a different storage performance requirements, then you can change the size and volume type here.
+
+![Choose Instance storage](images/aws/launch-4.png)
+
+Add tags. It is recommended you add as many tags as you need to correctly identify this server later. Especially if you have many servers, it can be difficult to remember which one was used for which purpose. At a very minimum, add a `Name` tag with something memorable.
+
+![Add tags](images/aws/launch-5.png)
+
+Configure security group. The security group is a stateful firewall that limits the traffic that is accepted by your instance. It is recommended to limit this as much as possible. In this case we are configuring it to only allow traffic on port 22 (ssh). It is recommended to restrict the source as well. To limit it to your workstation's current IP, click on the dropdown under "Source" and select "My IP".
+
+![Configure security group](images/aws/launch-6.png)
+
+Launch instance. Click on "Launch", and select a key pair that you will use to connect to the instance. If you already have a keypair that you have used previously, you can reuse it here by selecting "Choose an existing key pair". Otherwise you can create a keypair now by selecting "Create a new key pair".
+
+![Launch instance](images/aws/launch-7.png)
+
+Wait for instance to launch. When it is ready it will show as "running" under "Instance state", and "2/2 checks pass" under "Status Checks". Click on the instance to view the details on the bottom pane of the window, and look for the "IPv4 Public IP".
+
+![Wait for instance launch and view details](images/aws/ec2-list.png)
+
+Connect to instance. Using the keypair that you used or created in the launch step and the IP in the previous step, run ssh/
+
+![Connect to instance](images/aws/ssh-0.png)
+
+Run the following command in the SSH terminal. Note that there may be a more up to date version of the swift toolchain. Check https://swift.org/download/#releases for the latest available toolchain url for Amazon Linux 2.
+
+```
+SwiftToolchainUrl="https://swift.org/builds/swift-5.2.4-release/amazonlinux2/swift-5.2.4-RELEASE/swift-5.2.4-RELEASE-amazonlinux2.tar.gz"
+sudo yum install ruby binutils gcc git glibc-static gzip libbsd libcurl libedit libicu libsqlite libstdc++-static libuuid libxml2 tar tzdata ruby -y
+cd $(mktemp -d)
+wget ${SwiftToolchainUrl} -O swift.tar.gz
+gunzip < swift.tar.gz | sudo tar -C / -xv --strip-components 1
```

Comment:
IIUC this wil install the Swift toolchain, which means the host will run the compiler etc which could be a security a risk. we put together https://github.com/swift-server/guides/blob/main/packaging.md to describe different deployment strategies including a tarball, docker or source distribution (which this falls into IIUC). we probably want to update this to reflect these different strategies

Reply:
I guess the best choice now would be static stdlib linking, if only one Swift binary is deployed per host.

Reply:
+1